DAO.sol

```diff
@@ -586,7 +590,6 @@ contract DAO is DAOInterface, Token, TokenCreation {
             // related addresses. Proxy addresses should be forbidden by the curator.
             if (p.recipient != address(this) && p.recipient != address(rewardAccount)
                 && p.recipient != address(DAOrewardAccount)
-                && p.recipient != address(extraBalance)
                 && p.recipient != address(curator)) {
```

Comment:
In addition to my comment above. How do we actually move funds from the extraBalance to the DAO?

If a Proposal's recipient is the DAO itself, then the code in executeProposal() will send funds to itself when executing:

```
   if (!p.recipient.call.value(p.amount)(_transactionData))
      throw;
```

So it seems like we need some special code where the DAO can send a message to the extraBalance account by calling `extraBalance.payout(address(this), extraBalance.balance)`